perllib/FixMyStreet/App/Controller/Admin/ExorDefects.pm

```diff
@@ -140,6 +140,18 @@ sub download : Path('download') : Args(0) {
 
         foreach my $report (@$inspections) {
             my ($eastings, $northings) = $report->local_coords;
+
+            my $location = "${eastings}E ${northings}N";
+            $location = "[DID NOT USE MAP] $location" unless $report->used_map;
+            my $closest_address = $c->cobrand->find_closest($report);
+            if ($closest_address) {
+                # Match code in bin/oxfordshire/open311_service_request.cgi
```

Comment:
Yeah, I wondered about that; we can't/won't change the bin/oxfordshire code, so thought it might be better to match it, but it would be nicer to get some data back, you're right.